### PR TITLE
fix(kumactl): fix installation instructions

### PR DESCRIPTION
**Pull request by slonka**
Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? yes

### CODE REVIEW

**Review by lukidzi — APPROVED** [review]

File: app/_src/production/install-kumactl.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ and extract the archive with `tar -xvzf {{ site.mesh_helm_install_name }}-{{ pag
 
 Add the `kumactl` executable to your path:
 ```
-cd kuma-{{ page.version_data.version }}/bin
+cd {{site.mesh_product_name_lowercase}}-{{ page.version_data.version }}/bin
```
Comment (michaelbeaumont) [inline app/_src/production/install-kumactl.md:59]: does https://shopify.github.io/liquid/filters/downcase/ not work?

**Review by michaelbeaumont — CHANGES_REQUESTED** [review]

File: app/_src/production/install-kumactl.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ and extract the archive with `tar -xvzf {{ site.mesh_helm_install_name }}-{{ pag
 
 Add the `kumactl` executable to your path:
 ```
-cd {{site.mesh_product_name | downcase }}-{{ page.version_data.version }}/bin
+cd {{site.install_archive_name }}-{{ page.version_data.version }}/bin
```
Comment (lahabana) [inline app/_src/production/install-kumactl.md:59]: please use `mesh` as a prefix (useful for docs.konghq.com)

Comment (slonka) [reply app/_src/production/install-kumactl.md]: done

File: app/_src/production/install-kumactl.md (hunk 1 above)
Comment (slonka) [reply app/_src/production/install-kumactl.md]: it works but we're going with `mesh_install_archive_name`

**Review by michaelbeaumont — APPROVED** [review]

**Pull request merged**